pyFAI/ext/regrid_common.pxi

```diff
@@ -41,7 +41,18 @@ __license__ = "MIT"
 import cython
 import numpy
 import sys
-from libc.math cimport ceil, floor, copysign
+from libc.math cimport ceil, floor, copysign, pow, fabs, M_PI, sqrt, log, NAN
+
+cdef inline double pow2(double x) noexcept nogil:
+    return x*x
+
+cdef inline double pown(double x, unsigned int n) noexcept nogil:
+    cdef double result = 1.0
+    while n:
+        result *= x
+        n-=1
+    return result
```

Comment:
According to Cython's `cpow` compiler directive, this should not be needed:


Type of a | Type of b | cpow==True | cpow==False
-- | -- | -- | --
C floating point | C integer | Return type is floating point | Return type is floating point